x/ccv/consumer/keeper/genesis_test.go

```diff
@@ -390,6 +390,6 @@ func assertHeightValsetUpdateIDs(t *testing.T, ctx sdk.Context, ck *consumerkeep
 		require.Equal(t, heighValsetUpdateIDs[ctr].Height, height)
```

Comment:
nit: can you also name the `bool` callback output as `stop` above (line 389)

Reply:
Done